content/en/hosting-and-deployment/hosting-on-netlify.md

```diff
@@ -65,10 +65,10 @@ You can [set Hugo version](https://www.netlify.com/blog/2017/04/11/netlify-plus-
 
```

Comment:
To the best of my knowledge, Netlify abandoned their plans to support anything other than TOML.

Reply:
Wasn't aware. I will revert back to only TOML for Netlify-specific config.